snooty/rstspec.toml

```diff
@@ -58,6 +59,7 @@ ${4:   :align:
 """
 
 # Intended for use on the ecosystem landing page only
+# deprecated
```

Comment:
Do we want to raise a warning if this is used? If so, we can add a

`deprecated = true`

field to this directive

Reply:
@i80and as this flag raises a warning will it still parse correctly? This will still need to be useable as we need backward compatibility until `docs-ecosystem` latest version is released.

Reply:
It'll parse perfectly fine, it'll just log a warning